internal/pkg/cli/pipeline_update.go

```diff
@@ -272,6 +274,7 @@ func (o *updatePipelineOpts) Execute() error {
 		AppName:         o.appName,
 		Name:            pipeline.Name,
 		Source:          source,
+		Build:           build,
```

Comment:
nit: to remove the extra variable
```suggestion
		Build:           deploy.PipelineBuildFromManifest(pipeline.Build),
```